deploy/dependencies.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
In the suggested sequence of installation, dependencies.sh gets executed after docker images have been built. So having this docker installation command does not make sense here. It is better to execute this file immediately after copy.sh.